### DISCUSSION TITLE
New Installs

### DISCUSSION THREAD

**Discussion by danicrosby** [nss-evening-cohort-14/evening-client-side#117] [Discuss a Topic]
Do you want us to update prompts like these as they pop up? Or do we need to stay on the same versions of software we started in class. For example, I had Bootstrap v5 which was an issue for the webpack, so I downgraded to v4.6. Don't want install if it isn't required...

<img width="650" alt="Screen Shot 2021-02-18 at 12 54 45 PM" src="https://user-images.githubusercontent.com/68397076/108407074-df70e000-71e8-11eb-91a3-7f624bb508cf.png">

**Comment by ajawashington** [comment]
Yes, I would keep the current versions you have now and not do any updates. Unless Dr. T says otherwise ✨
